AdyenCard/Form/FormCardExpiryDateItem.swift

```diff
@@ -0,0 +1,63 @@
+//
+// Copyright (c) 2021 Adyen N.V.
+//
+// This file is open source and available under the MIT license. See the LICENSE file for more info.
+//
+
+import Adyen
+
+/// A form item into which card expiry date is entered, formatted and validated.
+internal final class FormCardExpiryDateItem: FormTextItem, Hidable {
```

Comment:
this is nice, extracting the logic in a separate class

Reply:
👍